src/server/index.js

```diff
@@ -64,7 +64,7 @@ app.prepare()
         logger.info("Set up notification queue and websocket");
         setUpAmqpForNotifications();
         expressWs(server);
-        server.ws(NavigationConstants.NOTIFICATION_WS, function(ws, request) {
+        server.ws(NavigationConstants.NOTIFICATION_WS, function (ws, request) {
```

Comment:
Is this from after running `prettier`?  I personally like the other look better 🤔

Reply:
Yes. I updated `prettier` from `1.x` to `2.x`. May be we could look into configuring `prettier` to format it the way it did in `1.x`

Reply:
I'm OK with the new default.
Here's a blurb about it: https://prettier.io/blog/2020/03/21/2.0.0.html#other-changes
Note that this is only for anonymous functions like this, and not something named like `function wsHandler(ws, request) { ... }`.